### PR TITLE
feat: last mod date

### PR DESCRIPTION
**Pull request by mshanemc**
### What does this PR do?
another SourceMember queried/stored field

### What issues does this PR fix or reference?
[@W-17371853@](https://gus.lightning.force.com/a07EE000026MwJ4YAK)

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -38,29 +38,6 @@ jobs:
...
-          - 'yarn test:nuts:deb'
-          - 'yarn mocha "test/nuts/trackingCommands/mpd-*.nut.ts" --slow 3000 --timeout 600000 --parallel'
-    with:
-      packageName: '@salesforce/source-tracking'
-      externalProjectGitUrl: 'https://github.com/salesforcecli/plugin-source'
-      command: ${{matrix.command}}
-      os: ${{matrix.os}}
-      preSwapCommands: 'yarn upgrade @salesforce/source-deploy-retrieve@latest; npx yarn-deduplicate; yarn install'
-      useCache: false
-    secrets: inherit
```
Comment (shetzel) [inline .github/workflows/test.yml:63]: 👍 good catch!

File: src/sourceTracking.ts [hunk 2]
```diff
@@ -330,7 +330,7 @@ export class SourceTracking extends AsyncCreatable {
       if (options.format === 'ChangeResultWithPaths') {
         return populateFilePaths({
           elements: filteredChanges.map(remoteChangeElementToChangeResult),
-          packageDirPaths: this.project.getPackageDirectories().map((pkgDir) => pkgDir.path),
+          packageDirPaths: this.project.getPackageDirectories().map((pkgDir) => pkgDir.fullPath),
```
Comment (shetzel) [inline src/sourceTracking.ts:333]: Existing tests don't seem to have a problem with this change, but I'm not clear why it's necessary and why new tests aren't needed.  Is this just relative versus absolute paths?

**Review by shetzel — APPROVED** [review]

File: src/sourceTracking.ts (hunk 2 above)
Comment (mshanemc) [reply src/sourceTracking.ts:333]: it's not in use by us.

https://github.com/search?q=org%3Asalesforcecli+ChangeResultWithPaths&type=code
https://github.com/search?q=org%3Aforcedotcom+ChangeResultWithPaths&type=code

I got one email about this, and I told them it wouldn't be a priority for CLI team to fix.  I started to use it for what I was building, fixed the bug, and then found a different better way, but decided to leave the bug fix in.

Comment (mshanemc) [reply src/sourceTracking.ts:333]: and yes, the fix is "use absolute path".  the problem occurs here https://github.com/forcedotcom/source-tracking/blob/8b05aa592a0c8025d18772d236d221915d34afb0/src/shared/populateFilePaths.ts#L67 when it calls ComponentSet.fromSource with a relative path and gets a file-not-found

**Pull request merged**